vendor/engines/c2po/app/views/facilities/_c2po_facility_fields.html.haml

```diff
@@ -0,0 +1,4 @@
+.well
+  %p= t('facilities.facility_fields.payment_options')
+  = f.input :accepts_po, :as => :boolean, :label => false, :inline_label => t('facilities.facility_fields.accepts_po')
+  = f.input :accepts_cc, :as => :boolean, :label => false, :inline_label => t('facilities.facility_fields.accepts_cc')
```

Comment:
I know this was pulled straight from the old partial but this could use style updating (quoting, hashrockets) before a merge.